crashes/index.html

```diff
@@ -349,7 +349,6 @@
     }
   });
   let baselines = [];
-  maxY = undefined;
```

Comment:
Hmm, I don't understand this. Why doesn't it error on the MG.data_graphic now that maxY isn't defined at all? (for the crash types not in THRESHOLD[channel]